README.md

```diff
@@ -8,8 +8,89 @@ User documentation for the SecureDrop Workstation.
 - Run `make docs` to start a live build of the documentation at `http://127.0.0.1:8000`
 - Edit RST files under the `docs` directory - your changes will be reflected in the live build
 
+## Detailed Setup instructions for macOS
+
+There are multiple ways to set your virtual environment; here we will use
```

Comment:
"we" vs "you" inconsistency. Probably best to keep things imperative/impersonal, like "The following procedure uses Homebrew and pyenv to set up a virtual environment."